docs/ui-setup.md

```diff
@@ -29,67 +29,77 @@ https://youtu.be/oR0I0aHxNMM?t=219
 
 This requires command line skills.
 
-### Starting the backend
 
-From the image above, download the `bitcoin-s-server` artifact.
-
-![Example banner](/img/installers.png)
-
-After unzipping, you will need to run `chmod +x ./bin/bitcoin-s-server` from the terminal make the file executable.
-
-After making the file executable, you can start the server with 
+First, clone the bitcoin-s-ts repository by running 
 
 ```
-unzip bitcoin-s-server.zip
-cd bitcoin-s-server
-chmod +x ./bin/bitcoin-s-server
-./bin/bitcoin-s-server
+git clone https://github.com/bitcoin-s/bitcoin-s-ts.git
 ```
 
-This starts the backend and will begin synchronizing with the bitcoin network.
-
+### Setup
 
-### Building and starting the frontend
+(If on a M1 Mac, go to the Generate M1 Mac server section below before continuing)
 
-Here are the instructions to build, you can paste this into your terminal
+You must have npm, if you don't then run `brew install node` 
+Next, navigate to the `bitcoin-s-ts` directory and in it run 
 
 ```
-git clone https://github.com/bitcoin-s/bitcoin-s-ts.git
-cd bitcoin-s-ts && npm i && run build
-cd wallet-server-ui && npm i && npm run build
-cd ../wallet-server-ui-proxy && npm i && npm run startlocal
+./build-wallet-electron.sh
 ```
+This will download the `bitcoin-s-server.zip` file and set up Suredbits Wallets dependencies. 
 
-The last command, `npm run startlocal` should result in server starting with logs that look like this
+Now, if you wish to run the Suredbits Wallet application in dev mode then run 
 
 ```
-> wallet-server-ui-proxy@1.9.0 startlocal
-> DEFAULT_UI_PASSWORD=none BITCOIN_S_SERVER_RPC_PASSWORD=password npx ts-node server.ts
-
-...
-
-ConfigureServerURL() http://localhost:9999/
-ConfigureAuthorizationHeader() Basic Yml0Y29pbnM6cGFzc3dvcmQ=
-[HPM] Proxy created: /  -> http://localhost:9999/
-[HPM] Proxy rewrite rule created: "^/api/v0" ~> ""
-[HPM] Proxy created: /  -> ws://localhost:19999/events
-[HPM] Proxy rewrite rule created: "^/websocket" ~> ""
-2022-03-11T17:05:19.238Z info: starting HTTP server
-2022-03-11T17:05:19.242Z info: Web Server started on port: 3002 ⚡
+cd wallet-electron-ts && npm run start"
+
 ```
+###Instill desktop application (Optional)
+
+If you would like to use the desktop application then run
 
-Now if you navigate to your web browser, you should see a page like this at `http://localhost:3002` 
+```
+npm make 
```

Comment:
is this right? or is it `npm run make`? Also isn't this run in `./build-wallet-electron.sh`

Reply:
Yeah it should be `npm run make`, my bad. I took the `npm run make` out of the code. Would you like me to put it back in so that the installer downloads and then give instructions on how to install i.e the part after `npm run make`?

Reply:
I don't understand why this is here if we put `npm run make` in `build-wallet-electron.sh`, won't that get run inside of that script?

Reply:
Yeah it will get run inside `build-wallet-electron.sh` but `npm run make` is not currently in the bash script would you like me to add it? Then I can take that step out of the instructions